src/model.js

```diff
@@ -2645,7 +2645,7 @@ class Model {
         }
       }
 
-      if (options.ignoreDuplicates && ['mssql', 'db2'].includes(dialect)) {
+      if (options.ignoreDuplicates && ['mssql', 'db2', 'oracle'].includes(dialect)) {
```

Comment:
Yes its accessable.  I see it can be added like this by checking if these properties are left default.
   ```
 if (options.ignoreDuplicates && this.sequelize.dialect.supports.inserts.ignoreDuplicates === '' &&
          this.sequelize.dialect.supports.inserts.onConflictDoNothing === '') {
```